hypha/apply/activity/adapters/slack.py

```diff
@@ -31,51 +31,55 @@ class SlackAdapter(AdapterBase):
     always_send = True
     messages = {
         MESSAGES.NEW_SUBMISSION: _(
-            "A new submission has been submitted for {source.page.title}: <{link}|{source.title}> by {user}"
+            "A new submission has been submitted for {source.page.title}: <{link}|{source.title_text_display}> by {user}"
         ),
         MESSAGES.UPDATE_LEAD: _(
-            "The lead of <{link}|{source.title}> has been updated from {old_lead} to {source.lead} by {user}"
+            "The lead of <{link}|{source.title_text_display}> has been updated from {old_lead} to {source.lead} by {user}"
         ),
         MESSAGES.BATCH_UPDATE_LEAD: "handle_batch_lead",
         MESSAGES.COMMENT: _(
             "A new {comment.visibility} comment has been posted on <{link}|{source.title}> by {user}"
```

Comment:
Should this be:
```suggestion
            "A new {comment.visibility} comment has been posted on <{link}|{source.title_text_display}> by {user}"
```
That's only one I noticed that was missing on slack